nullaway/src/main/java/com/uber/nullaway/ErrorBuilder.java

```diff
@@ -85,9 +92,13 @@
    * @return the error description
    */
   Description createErrorDescription(
-      ErrorMessage errorMessage, Description.Builder descriptionBuilder, VisitorState state) {
+      ErrorMessage errorMessage,
+      Description.Builder descriptionBuilder,
+      VisitorState state,
+      Symbol fixable) {
```

Comment:
`fixable` is not a good name.  How about `nonNullTarget`?  Meaning that if non-null, this error involved a pseudo-assignment of a `@Nullable` expression into a `@NonNull` target, and this parameter is the `Symbol` for that target.  Also please update the Javadoc.  Also, please mark the parameter as `@Nullable`

Reply:
Changed it. f2c044fb80078bff8f8015e59c7815a2f8149609